qiskit_alice_bob_provider/plugins/sk_synthesis.py

```diff
@@ -135,25 +136,25 @@ def pass_manager(
         # SolovayKitaevSynthesis plugin) with the gates to synthesize and basis
         # gates computed above.
         if pass_manager_config.unitary_synthesis_method == 'sk':
-            for passess in pm.passes():
-                for passes in passess.values():
-                    for p in passes:
-                        if (
-                            isinstance(p, UnitarySynthesis)
-                            and p.method == 'sk'
-                        ):
-                            # There is no option to manually set the gates to
-                            # synthesize in UnitarySynthesis, and the default
-                            # _synth_gates is just 'unitary'!
-                            # pylint: disable=protected-access
-                            p._synth_gates = synth_gates
-                            # Can't pass basis gates in
-                            # unitary_synthesis_plugin_config because overriden
-                            # by global basis_gates.
-                            # This seems to be a Qiskit bug (why give the
-                            # possibility to specify basis gates in the plugin
-                            # config if that was not the intent?)
-                            # pylint: disable=protected-access
-                            p._basis_gates = discrete_basis_gates
+            # pylint: disable=protected-access
+            for task in pm._tasks:
```

Comment:
It seems to me this was done between [0.44](https://docs.quantum.ibm.com/api/qiskit/0.44/qiskit.transpiler.PassManager#passes) and [0.45](https://docs.quantum.ibm.com/api/qiskit/0.45/qiskit.passmanager.BasePassManager). I can't find a reference to it in [the release notes](https://docs.quantum.ibm.com/api/qiskit/release-notes/0.45), do you know if there is an alternative to `.passes()`, to avoid relying on a private attribute ?

Reply:
Ok I missed the code below, already relying on private methods. I guess that if we have issues with this at some point we'll need to find a better workaround for the whole SK approximation